wls-ergebnismeldung-service/src/main/java/de/muenchen/oss/wahllokalsystem/ergebnismeldungservice/service/awerte/AWerteValidator.java

```diff
@@ -0,0 +1,21 @@
+package de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.service.awerte;
+
+import de.muenchen.oss.wahllokalsystem.ergebnismeldungservice.exception.ExceptionConstants;
+import de.muenchen.oss.wahllokalsystem.wls.common.exception.util.ExceptionFactory;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class AWerteValidator {
+
+    private final ExceptionFactory exceptionFactory;
+
+    public void validWahlbezirkIDParamOrThrow(final String wahlbezirkID) {
+        if (wahlbezirkID == null || StringUtils.isBlank(wahlbezirkID) || StringUtils.isEmpty(wahlbezirkID)) {
```

Comment:
```suggestion
        if (StringUtils.isBlank(wahlbezirkID)) {
```
Ich glaube `wahlbezirkID == null` und `StringUtils.isEmpty(wahlbezirkID)` ist in `StringUtils.isBlank(wahlbezirkID)` bereits enthalten

Reply:
resolved with d8bc6d266f1a8698d28078ce997a51f2bf37b5f4